group_vars/moc/vlans.yaml

```diff
@@ -0,0 +1,940 @@
+vlans:
+  10:
+    name: "CSAIL-MAIN"
+    description: "Various Openshift clusters"
+  105:
+    name: "MOC-BU-PUBLIC"
+    description: "Public IPs from BU"
+  127:
+    name: "MOC-NEU-PUBLIC"
+    description: "NEU Public IP for infrastructure"
+  204:
+    name: "MOC-GENERAL"
+    description: "Intranet (routable to internet). SNMP for client nodes."
+  208:
+    name: "ESI-CONTROL-PLANE"
+    description: "ESI control plane"
+  211:
+    name: "MOC-NESE"
+    description: "New England Storage Exchange (NESE)"
+  213:
+    name: "MOC-PROD-CEPH-ISCSI"
+    description: "Ceph Cluster iSCSI"
+  214:
+    name: "MOC-NEW-ESI"
+    description: "New ESI deployment"
+  250:
+    name: "MOC-PROD-CEPH"
+    description: "Ceph public (for clients)"
```

Comment:
VLAN 277 and 280 should be here - those are managed by MaaS. 

Not sure if you skipped 278 on purpose (iventoy)

Reply:
I skipped iventoy on purpose, we can circle back if we ever end up using it